zcncore/transactionauth_base.go

```diff
@@ -13,31 +13,31 @@ import (
 )
 
 type TransactionWithAuth struct {
-	t *Transaction
+	*Transaction
```

Comment:
@dabasov FYI - went with the struct embedding simplification with this same pr (not on a different PR) to save us time on implementing the same methods of `Transaction` in `TransactionWithAuth` over and over again.

Ref: https://github.com/0chain/gosdk/blob/7990ac1c9f6179c052d09583a906c171a542cefe/zcncore/transactionauth_base.go#L89-L93